### PR TITLE
Added fixes to clear on close and component superposition

### PR DESCRIPTION
**Pull request by arthur911016**

### CODE REVIEW

File: vanillaSelectBox.css [hunk 1]
```diff
@@ -25,6 +25,9 @@ li[data-parent].open:not(.hidden-search){
   border-radius:4px;
   font-size : 11px;
 }
+.vsb-menu.open-to-top {
+	box-shadow: 0 -6px 12px rgba(0,0,0,.175);
```
Comment (jaguerra2017) [inline vanillaSelectBox.css]: code style

File: vanillaSelectBox.js [hunk 2]
```diff
@@ -265,6 +265,13 @@ function vanillaSelectBox(domSelector, options) {
 
         this.rootToken = self.domSelector.replace(/[^A-Za-z0-9]+/, "")
         this.root.style.display = 'none';
+        let holder = document
```
Comment (jaguerra2017) [inline vanillaSelectBox.js]:
```
let holder =
   document.que....
```

File: vanillaSelectBox.js [hunk 3]
```diff
@@ -618,29 +631,36 @@ function vanillaSelectBox(domSelector, options) {
...
+                    window.addEventListener('mousewheel', self.handleScroll);
+                    window.addEventListener('mousedown', self.handleMouseDown);
+                    document.addEventListener("click", docListener);
+                    e.preventDefault();
+                    e.stopPropagation();
+                    if(!self.userOptions.stayOpen ){
+                        VSBoxCounter.closeAllButMe(self.instanceOffset);
+                    }
+				});
+		}
```
Comment (jaguerra2017) [inline vanillaSelectBox.js]: multiple wrong indent

**Review by jaguerra2017 — CHANGES_REQUESTED** [review]

File: vanillaSelectBox.js [hunk 4]
```diff
@@ -639,28 +639,28 @@ function vanillaSelectBox(domSelector, options) {
...
+            self.drop.style.boxShadow = "none";
+            self.drop.style.minHeight =  (this.userOptions.maxHeight+10) + "px";
+            self.drop.style.position = "relative";
+            self.drop.style.left = "0px";
+            self.drop.style.top = "0px";
+            self.button.style.border = "none";
+        } else {
+            this.main.addEventListener('click', function (e) {
+                if (self.isDisabled) return;
+                self.drop.style.display = "block";
```
Comment (jaguerra2017) [inline vanillaSelectBox.js]: 2 spaces indent

Comment (jaguerra2017) [reply vanillaSelectBox.js]: fix across the PR

**Review by jaguerra2017 — CHANGES_REQUESTED** [review]

File: vanillaSelectBox.css [hunk 5]
```diff
@@ -25,6 +25,9 @@ li[data-parent].open:not(.hidden-search){
   border-radius:4px;
   font-size : 11px;
 }
+.vsb-menu.open-to-top {
+	box-shadow: 0 -6px 12px rgba(0, 0, 0, .175);
```
Comment (jaguerra2017) [inline vanillaSelectBox.css]: wrong indent

**Review by jaguerra2017 — CHANGES_REQUESTED** [review]

File: vanillaSelectBox.css (hunk 5 above)
Comment (arthur911016) [reply vanillaSelectBox.css]: Fixed the css file changed intent to 4 spaces like the .js file

**Pull request merged**